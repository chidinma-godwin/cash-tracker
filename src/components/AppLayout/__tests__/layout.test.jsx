import { render } from '@testing-library/react';
import Layout from '../layout';

describe('Layout', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(
      <Layout>
        <div className='children' />
      </Layout>
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
