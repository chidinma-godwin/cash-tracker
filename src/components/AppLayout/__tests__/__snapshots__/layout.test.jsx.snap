// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="makeStyles-root-2"
  >
    <header
      class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4"
    >
      <div
        class="MuiContainer-root MuiContainer-maxWidthLg"
      >
        <div
          class="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
        >
          <h5
            class="MuiTypography-root makeStyles-title-3 MuiTypography-h5"
          >
            <a
              class="makeStyles-linkStyle-4"
              href="/"
            >
              Cash Tracker
            </a>
          </h5>
          <button
            aria-haspopup="true"
            aria-label="account of current user"
            class="MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeEnd"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiIconButton-label"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z"
                />
              </svg>
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </header>
  </div>
  <div
    class="MuiGrid-root MuiGrid-container"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-md-2"
    />
    <div
      class="MuiGrid-root makeStyles-gridRoot-1 MuiGrid-item MuiGrid-grid-xs-10 MuiGrid-grid-md-8"
    >
      <div
        class="children"
      />
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-1 MuiGrid-grid-md-2"
    />
  </div>
</DocumentFragment>
`;
